sabnzbd/downloader.py

```diff
@@ -764,6 +764,18 @@ def process_nw(self, nw: NewsWrapper):
                 nw.reset_data_buffer()
                 self.__request_article(nw)
 
+            else:
+                logging.warning(
```

Comment:
Could you enclose the warning in the T function like I showed in my comment? That's needed to make the text translatable for other languages. And  I think we should not include the nntp_msg as this could be a huge binary blob if somehow article data was included.

Reply:
I had somehow missed your previous comment entirely.

I didn't wrap it in the `T()` function as nothing else in that file is wrapped.

For the nntp_msg can we just include the first line? The whole line may provide a context-clue as to why the server responded how it did and how it should be handled in future (eg 451 is `451 0 <articleid> cancelled`)

Reply:
> I didn't wrap it in the `T()` function as nothing else in that area of the file was wrapped.

Only warnings/errors need that, since those are shown to the user. This is a warning, so it needs it.

> For the nntp_msg can we just include the first line? The whole line may provide a context-clue as to why the server responded how it did and how it should be handled in future (eg 451 is `451 0 <articleid> cancelled`)

In that case we will just have to manually debug it by connecting to that server. 
No need to complicate this, just the code will be enough for now :)

Reply:
This is now wrapped with marked as translated in `9256c87e657ea0202fd717f6d6b4f9859c8192b3`, and I've made it only show the first line of the output, which will be the header line.